tests/unit/test__signing.py

```diff
@@ -577,7 +577,7 @@ def _generate_helper(
         )
         self.assertEqual(scheme, expected_scheme)
         self.assertEqual(netloc, expected_netloc)
-        self.assertEqual(path, resource)
+        self.assertEqual(path, six.moves.urllib.parse.quote(resource))
```

Comment:
Does this encode '/' as well? It should not.

Reply:
It does not:  it is actually designed to quote the path part.

Reply:
This might hit an issue with conformance tests when that's added but approving for now.

Reply:
As it happens, I backed it out here, because the bucket / blob already do the urlencoding of the resource before calling `_sigining.generate_signed_url_v4`.